lib/workers/branch/index.ts

```diff
@@ -320,6 +326,83 @@ export async function processBranch(
     } else {
       logger.debug('No updated lock files in branch');
     }
+
+    if (global.trustLevel === 'high') {
```

Comment:
```suggestion
    if (global.trustLevel === 'high' && 
      is.nonEmptyArray(config.allowedPostUpgradeCommands)) {
```

Sorry, it might need prettier-fying too